ci/kind/kind-setup.sh

```diff
@@ -663,7 +663,7 @@ while [[ $# -gt 0 ]]
       shift 2
       ;;
     --flexible-ipam)
-      add_option "--flexible-ipam" "create"
+      add_option "--flexible-ipam" "$ACTION"
```

Comment:
If `--flexible-ipam` can be sued with both actions, you can skip calling `add_option`. IIRC, `add_option` is only meant for option validation (i.e., is this option compatible with this action).